.evergreen.yml

```diff
@@ -12448,6 +12516,7 @@ tasks:
           executable_os_id: win32
       - func: run_e2e_tests
         vars:
+          task_name_var: "${task_name}"
```

Comment:
Keep in mind that this file is automatically generated from `.evergreen/evergreen.yml.in` 🙂

Reply:
Oh! so are my changes not doing anything at all? I have suspicion that `${task_name}` is always empty right now for the `run_e2e_tests` and have been battling it for so long

Reply:
```
npm run update-evergreen-config
```
oh cool!

Reply:
Yeah, although any changes you make to `.evergreen.yml` should already have an effect (they'd just get overwritten after merging).

I don't know what exactly's going on, but you shouldn't need the `task_name_var` indirection here, using `${task_name}` directly in the `env:` list should already work

Reply:
Yeah, I am suspecting this expansion is either not working or somehow getting lost in the process. I looked through evergreen docs and nothing suspicious there.

Reply:
Okay seems like the env variable just somehow doesn't reach the bash script